metric_learn/lmnn.py

```diff
@@ -224,6 +224,9 @@ def fit(self, X, y):
 
     it = 1  # we already made one iteration
 
+    if self.verbose:
+      print("it | objective | delta_obj | total_active | learn_rate")
```

Comment:
maybe give more explicit names? some suggestions:
it --> iter or iteration
delta_obj --> objective difference
total_active --> active constraints
learn_rate --> learning rate